http/src/main/resources/reference.conf

```diff
@@ -110,7 +110,7 @@ dataproc {
 }
 
 gce {
-  customGceImage = "projects/broad-dsp-gcr-public/global/images/leo-gce-image-2024-02-14-15-02-28"
+  customGceImage = "projects/broad-dsp-gcr-public/global/images/leo-gce-image-2024-04-24-12-44-28"
```

Comment:
Oh, I see, there is no need to do that FYI since the GCE image has not changed, but it does not hurt